pkg/backup/resource_backupper.go

```diff
@@ -174,7 +176,7 @@ func (rb *defaultResourceBackupper) backupResource(
 	)
 
 	var namespacesToList []string
-	if resource.Namespaced {
+	if resource.Namespaced || resource.Kind == "Namespace" {
```

Comment:
As we discussed on Slack, this doesn't do what you think it might. If you call `rb.dynamicFactory.ClientForGroupVersionResource(gv, resource, namespace)` where `resource` is the namespaces resource, and `namespace` is a specific name of a namespace, the call that immediately follows:

```go
unstructuredList, err := resourceClient.List(metav1.ListOptions{LabelSelector: rb.labelSelector})
```

will still return a full listing of all of the namespaces.